apps/judicial-system/web/src/utils/validate.ts

```diff
@@ -297,10 +302,65 @@ export const isProcessingStepValidIndictments = (
 export const isTrafficViolationStepValidIndictments = (
   workingCase: Case,
 ): boolean => {
-  return Boolean(
+  const hasValidDemands = Boolean(
     workingCase.demands &&
       (!workingCase.hasCivilClaims || workingCase.civilDemands),
   )
+
+  if (!workingCase.indictmentSubtypes || !hasValidDemands) {
+    return false
+  }
+
+  const validateIndictmentCount = (indictmentCount: IndictmentCount) =>
```

Comment:
Niiit: since this returns a boolean what do you think about referring to this as `isValidIndictmentCount` instead of `validateIndictmentCount`? Since `validateIndictmentCount` might indicate a simple `void` func for example?

Reply:
Yes, done ✔️ Thanks..